servers/republik/script/prolong/paperInvoice.js

```diff
@@ -120,9 +210,10 @@ PgDb.connect().then(async pgdb => {
     log(csvFormat(paperPeopleNeedProlong.filter(d => d.address).map(mapToCsv)))
     log('---')
   }
+
+  await pgdb.close()
+
   log('finished!')
-}).then(() => {
-  process.exit()
```

Comment:
at least for me the script doesn't stop now!

Reply:
It should not kill the process before `console.log` has printed everything in buffer.
It did before when print CSV lines.

Quitting job manually was easiest.
Graceful quit e.g. with an interval on SIG* would be better.
Use `process.stdout` probably best.